core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -143,7 +186,6 @@ public BigQuerySqlDialect(SqlDialect.Context context) {
 
   private static final SqlSetOperator INTERSECT_DISTINCT =
       new SqlSetOperator("INTERSECT DISTINCT", SqlKind.INTERSECT, 18, false);
-
```

Comment:
undo this change

Reply:
Done. I think I should check `git diff` more frequently.